app/models/container_node.rb

```diff
@@ -72,17 +73,35 @@ def perf_rollup_parents(interval_name = nil)
     [ext_management_system] unless interval_name == 'realtime'
   end
 
-  def ipaddress
+  def kubernetes_hostname
     labels.find_by(:name => "kubernetes.io/hostname").try(:value)
   end
 
   def cockpit_url
-    URI::HTTP.build(:host => ipaddress, :port => 9090)
+    URI::HTTP.build(:host => kubernetes_hostname, :port => 9090)
   end
 
   def evaluate_alert(_alert_id, _event)
     # currently only EmsEvents from hawkular are tested for node alerts,
     # and these should automaticaly be translated to alerts.
     true
   end
+
+  supports :external_logging_support do
+    unless ext_management_system.respond_to?(:external_logging_route_name)
+      unsupported_reason_add(:external_logging_support, _('This provider type does not support Extrenal Logging'))
+    end
+  end
+
+  def external_logging_query
+    nil # {}.to_query # TODO
+  end
+
+  def external_logging_path
+    fqdn = (kubernetes_hostname || name || "")
+    hostname = fqdn.split('.').first || ""
+    query = "bool:(filter:(or:!((term:(hostname:'#{hostname}')),(term:(hostname:'#{fqdn}')))))"
```

Comment:
@simon3z I want a query to get all the message with hostname that equals to the fqdn of the node OR the hostname of the node. This is derived from looking at the values of this field in the logs that are currently being collected.
I have tested that if either `fqdn` or `hostname` will be empty string `""` it will not add results to the search query results.